package insight_msgs;

message RobotDetection
{
    required double bearing = 1;
    required double range = 2;
    required double robot_x = 3;
    required double robot_y = 4;
    optional int32 red = 5;
    optional int32 green = 6;
    optional int32 blue = 7;
}